src/Experimental/ShortestPaths/ShortestPaths.jl

```diff
@@ -99,12 +98,17 @@ s4 = shortest_paths(g, 1, BellmanFord())
 s5 = shortest_paths(g, 1, w, DEsopoPape())
 ```
 """
-shortest_paths(g::AbstractGraph, s, alg::ShortestPathAlgorithm) =
-    shortest_paths(g, s, weights(g), alg)
+function shortest_paths(g::AbstractGraph, s, alg::ShortestPathAlgorithm)
+    return shortest_paths(g, s, weights(g), alg)
+end
```

Comment:
This transformation is specified by blue - do we want that?

Reply:
I think so, yes. When the lines are too long, Blue replaces
```julia
f(x) = y
```
with
```julia
function f(x)
    return y
end
```